internal/cmd/tui.go

```diff
@@ -133,7 +133,14 @@ func tuiCmd() *cobra.Command {
 						return err
 					}
 
-					err = runnerClient.RunBlock(ctx, result.block)
+					tempBlock := result.block.Block.Clone()
+
+					err = promptEnvVars(cmd, []project.FileCodeBlock{&tempBlock})
```

Comment:
You should take this code block out of `inRawMode` since I don't think bubbletea is meant to run in raw mode.